src/frontend/src/components/Booking/BookingCard.tsx

```diff
@@ -0,0 +1,216 @@
+            <p className="font-semibold text-stone-400">
+              {options.props.gallery.location}
+            </p>
+          </div>
+        </div>
+        <div className="col-span-2 flex gap-x-2 pl-4">
+          <div className="relative w-8 h-8 rounded-full">
+            <Image
+              className="object-cover rounded-full"
+              fill={true}
+              src={
+                customer?.profile_picture_url
+                  ? customer.profile_picture_url
+                  : "/images/no-picture.jpeg"
+              }
+              alt=""
+            />
+          </div>
+          <div className="">
+            <p className="text-sm font-semibold">
+              {customer ? customer.data.firstname : ""}{" "}
+              {customer ? customer.data.lastname : ""}
+            </p>
+            <p className="text-sm font-semibold text-stone-400">
+              @{customer ? customer.data.firstname : ""}
+            </p>
+          </div>
+        </div>
+        <div className="col-span-2 pl-4">
+          <p className="text-base font-semibold flex gap-x-2">
+            <svg
+              width="19"
+              height="22"
+              viewBox="0 0 19 22"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                d="M6.33333 6.41667V2.75M12.6667 6.41667V2.75M5.54167 10.0833H13.4583M3.95833 19.25H15.0417C15.4616 19.25 15.8643 19.0568 16.1613 18.713C16.4582 18.3692 16.625 17.9029 16.625 17.4167V6.41667C16.625 5.93044 16.4582 5.46412 16.1613 5.1203C15.8643 4.77649 15.4616 4.58333 15.0417 4.58333H3.95833C3.53841 4.58333 3.13568 4.77649 2.83875 5.1203C2.54181 5.46412 2.375 5.93044 2.375 6.41667V17.4167C2.375 17.9029 2.54181 18.3692 2.83875 18.713C3.13568 19.0568 3.53841 19.25 3.95833 19.25Z"
+                stroke="#0C0A09"
+                strokeWidth="2"
+                strokeLinecap="round"
+                strokeLinejoin="round"
+              />
+            </svg>
+            {transformDate(options.props.start_time).day}/
+            {transformDate(options.props.start_time).month}/
+            {transformDate(options.props.start_time).year}
+          </p>
+          <p className="text-base font-semibold">
+            {transformDate(options.props.start_time).hour}
+            {"."}
+            {transformDate(options.props.start_time).minute}
+            {" - "}
+            {transformDate(options.props.end_time).hour}
+            {"."}
+            {transformDate(options.props.end_time).minute}
+          </p>
+          <p className="text-xs font-semibold text-stone-400">
+            15 days from now
+          </p>
+        </div>
+        <div className="text-base font-bold pl-4">
+          <span>{options.props.gallery.price} THB</span>
+        </div>
+        <div className="col-span-2 pl-4">
+          <div className="text-base font-bold">
+            {options.props.status == BookingStatus.BookingCancelledStatus ? (
```

Comment:
Consider using switch case to possibly achieve faster response time and cleaner code kub